util/embeddings/kms.py

```
@@ -0,0 +1,126 @@
+    @property
+    def model_id(self) -> str:
+        """Return the base generator's model identifier."""
+        return self._base.model_id
+
+    def generate(
+        self,
+        text: str,
+        concept_type: str | None = None,
+        attribute: str | None = None,
+        span: Any | None = None,
+        metadata: dict[str, Any] | None = None,
```

Ruff (0.14.13)

Unused method argument: `metadata`

(ARG002)